numpy/core/tests/test_multiarray.py

```diff
@@ -9688,17 +9688,3 @@ def test_sort_uint():
     arr = rng.integers(low=0, high=maxv, size=N).astype('uint32')
     arr[np.random.choice(arr.shape[0], 10)] = maxv
     assert_equal(np.sort(arr, kind='quick'), np.sort(arr, kind='heap'))
-
-
-@pytest.mark.parametrize("a",
-    (
-        np.array(2),  # 0D array
-        np.array([3, 2, 7, 0]),  # 1D array
-        np.arange(6).reshape(2, 3),  # 2D array
-    )
-)
-def test__fastCopyAndTranspose(a):
-    from numpy.core.multiarray import _fastCopyAndTranspose
-    b = _fastCopyAndTranspose(a)
```

Comment:
nit: maybe leave the test and add a warning filter, and check that the filter catches a warning?